parser_test.go

```diff
@@ -1,44 +1,101 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
+	"image"
+	"image/color"
+	"image/draw"
+	"image/jpeg"
 	"os"
+	"reflect"
 	"testing"
+
+	wa "github.com/Rhymen/go-whatsapp"
 )
 
 const (
-	envTestURL1 = "TEST_URL_1"
-	envTestURL2 = "TEST_URL_2"
-	envTestURL3 = "TEST_URL_3"
+	testBaseURL = "http://127.0.0.1:6969"
 )
 
 var (
-	testURL1, testURL2, testURL3 string
+	testTextURL1  = testBaseURL + "/text/1"
+	testTextURL2  = testBaseURL + "/text/2"
+	testImageURL1 = testBaseURL + "/image/1"
 )
 
 func TestNemoParser(t *testing.T) {
```

Comment:
change the testing process
- see https://github.com/eFishery/NeMo/pull/12#issue-499060846 on **How Has This Been Tested?** for more details
- edit cmd/testserver/main.go as needed